docs/docs/eslint.md

```diff
@@ -1,33 +1,33 @@
 ---
-title: Using ESLint
+título: Utilización ESLint
```

Comment:
"Usando ESLint" mejor?

Reply:
Hi, I just finished resolving this comment.